bin/fixme

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
 
-var FixMe = require('../index');
+var FixMe = require('../lib/fixMe');
```

Comment:
I think generally all filenames should be underscored (but the variable name looks good)